src/button.js

```diff
@@ -0,0 +1,57 @@
+class Button {
+  constructor(
+    text,
+    width,
+    lang = 'en',
+    altText,
+    code
+  ) {
+    this.keyNode = document.createElement('button');
+    this.text = text;
+    this.width = width;
+    this.lang = lang;
+    this.altText = altText;
```

Comment:
```suggestion
    this.altText = altText ? altText : text;
```
For avoiding duplicates in keyConfig.js

Reply:
Thank you! Fix